UnitsNet/UnitDefinitions/FoulingFactor.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Name": "FoulingFactor",
+  "BaseUnit": "HourSquareFeetDegreefahrenheitPerBtu",
+  "XmlDoc": "Heat Transfer Coefficient or Thermal conductivity - indicates a materials ability to conduct heat.",
+  "Units": [
+    {
+      "SingularName": "HourSquareFeetDegreefahrenheitPerBtu",
```

Comment:
The `hr` / `Hour` time unit seems out of place here? But I am not familiar with these units.